specification/containerservice/resource-manager/Microsoft.ContainerService/aks/preview/2023-06-02-preview/managedClusters.json

```diff
@@ -6733,6 +6733,10 @@
           "$ref": "#/definitions/ManagedClusterSecurityProfileImageCleaner",
           "description": "Image Cleaner settings for the security profile."
         },
+        "imageIntegrity": {
+          "$ref": "#/definitions/ManagedClusterSecurityProfileImageIntegrity",
+          "description": "Image integrity is a feature that works with Azure Policy to verify image integrity by signature. This will not have any effect unless policy is applied to enforce image signatures. See https://aka.ms/aks/image-integrity for how to use this feature via policy."
```

Comment:
policy -> Azure Policy?

or don't mention it at all, since in API you're to check the dependency.

Reply:
changed policy to Azure Policy
Will choose to mention it since we want to avoid customer to enable image integrity directly by arm as possible

Reply:
This is API spec, just describe whatever the API used for.

No need to impact user behavior.. `avoid customer to enable image integrity directly by arm`. As allow as it is allowed, it is supported. If you want some add hint, that should be in doc.

Reply:
This does show up in documentation in SDKs and REST API docs @karataliu, so I think it's appropriate to say that you must use policy with this here too? Just my 2c though.

Reply:
sounds good to keep it. Can we unify all occurrences of `Azure Policy` term?
It was referred as `Azure Policy` then `policy`, `Azure Policy` sounds like a proper noun, but then `policy` is like a casual word. I'd suggest always use `Azure Policy` or make it `Azure policy` in first occurrence (then it shows. `policy` is pointing to same thing)